src/test/java/MyStepdefs.java

```diff
@@ -9,6 +9,7 @@ public class MyStepdefs {
 
     public void iPerformTheGETOperationFor() {
         given().contentType("");
-        System.out.println("Just an example");
+        System.out.println("This is the nre change on April 9");
```

Comment:
remove printing statement

Reply:
@saranyanight99  addresed the code changes suggested
please review and approve